src/SamplesApp/SamplesApp.Wasm/SamplesApp.Wasm.csproj

```diff
@@ -69,7 +69,6 @@
 		<PackageReference Include="Microsoft.Identity.Client" Version="4.47.0" />
 		<PackageReference Include="MSTest.TestFramework" Version="2.1.2" />
 		<PackageReference Include="Uno.Extensions.Logging.WebAssembly.Console" Version="1.4.0" />
-		<PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
```

Comment:
We should be getting this from the BCL. No need to add a package reference with an old version.